src/SearchWrapper.cpp

```diff
@@ -140,7 +140,7 @@ Rcpp::List searchWrapper(Rcpp::CharacterVector & dbpath,Rcpp::List & queryList){
   
   int cols=0;
   const string & data = mset.begin().get_document().get_data();
-  cols=columnNo(data);
+  cols=lineCount(data); //data consists of row values of a number of columns in indexed data frame seperated by \n
```

Comment:
I'd be inclined to not have the comment here, because not everyone who needs to know is going to look here for it. It needs to go somewhere, though, maybe in the comment for `indexWrapper()` (which is what creates this structure).